cypress/integration/seededFlows/articleFlows/liquidTagFlows/userSubscriptionLiquidTagWithAppleAuthEmail.spec.js

```diff
@@ -0,0 +1,43 @@
+describe('User subscription liquid tag - apple auth email', () => {
```

Comment:
I've separated the two user cases (apple auth / non apple auth) into separate test files, purely because we've had such headaches with any Cypress tests that require switching users between different tests.